src/components/atoms/ControlledFields/DateTimePicker/types.ts

```diff
@@ -0,0 +1,10 @@
+// import { Dayjs } from 'dayjs';
+import { Control, Path } from 'react-hook-form';
+
+export interface IProps<T extends object> {
+  control: Control<T, object>;
+  name: Path<T>;
+  label?: string;
+  // minDate?: Dayjs;
```

Comment:
same

Reply:
the same here. I'll add an additional comment to this file in order other developers will have more context